src/layouts/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import EmailIcon from "../assets/logo/envelope.svg";
+import FacebookIcon from "../assets/logo/facebook.svg";
+import InstagramIcon from "../assets/logo/instagram.svg";
+
+function Footer() {
+  return (
+    <footer className="w-full bg-[var(--color-gray-10)] py-6 px-4 gap-2">
```

Comment:
```suggestion
    <footer className="w-full bg-gray-10 py-6 px-4 gap-2">
```
`var(--color-gray-10)` CSS 변수는 이미 수민님께서 `theme 변수로지정해두셔서
위와 같이 사용할 수 있습니다 👍

아래 Vscode Extension을 활용하시면 자동완성으로 사용할 수 있는 클래스명을 알 수 있어 편리해요!
![image](https://github.com/user-attachments/assets/20c911d5-6029-4a46-a1ff-82ff2c7e9877)